main.rb

```diff
@@ -0,0 +1,12 @@
+$file_directory = "games.log"
+class Main
+    def readFile
```

Comment:
funções e váriaveis são declaradas no formato `snake_case`
```suggestion
    def read_file
```